### PR TITLE
Better error handling

### PR DESCRIPTION
**Pull request by vladbalanescu**
Changes:
---
- [x] better error handling in case of missing labels

Bugs:
---
Part of https://github.com/PeterNgTr/codeceptjs-testrail/issues/40

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ class TestRail {
 			output.log(`The response is ${JSON.stringify(res.data)}`);
 			return res.data;
 		}).catch(error => {
-			output.error(`Cannot add result for case due to ${error}`);
+			output.error(`Cannot add result for case due to ${error.response.data.error}`);
```
Comment (Georgegriff) [inline index.js:125]: might want to consider being a bit defensive here just incase error.response.data is undefined or error.response.data.error is undefined too

Comment (vladbalanescu) [reply index.js]: sorted, cheers

**Comment by vladbalanescu** [comment]
The above change would result in an output similar to this one:
```
15:23:26  tests-runner | Cannot add result for case due to Field :results contains one or more invalid results (case C362669 unknown or not part of the test run)
```

**Comment by vladbalanescu** [comment]
@PeterNgTr Any chance you can have a look over this one? It would clearly help others in terms of more verbose logs on error cases 🙃

**Comment by PeterNgTr** [comment]
That's great! Thanks for deep dive in it.

**Comment by PeterNgTr** [comment]
So @vladbalanescu @Georgegriff I've published new version with your improvement. Thanks.

**Comment by vladbalanescu** [comment]
absolutely brilliant, thanks a lot!

**Pull request merged**